### ISSUE TITLE
ps-form: ps-form__savebar has margin

### ISSUE THREAD

**Issue by sdiermeier** [prosoft-edv/components#131]
The savebar has a margin which is not needed/should be removed

![image](https://user-images.githubusercontent.com/19778499/93991160-40a49c80-fd8c-11ea-903e-6d234d019d5e.png)

**Comment by saithis** [comment]
Yes, the margin should be changed to "margin-top: 1em". But this is a breaking change... so should we maybe make an update schematic, which adds "display:block; margin-bottom: 1em;" to every ps-form? This could still be breaking, if the ps-form already has a conflicting style, bit this is less likely. Or should we just change the margin and fix every usage by hand? 

@Yirica what do you think?

@sdiermeier can you please make a PR for this, after this is clarified?
